templates/index.html

```diff
@@ -2,10 +2,11 @@
 
 {% block content %}
 {% if current_user.is_authenticated %}
-    <div class='some-page-wrapper'>
+    <div class='{{ config["DASHBOARD_PAGE_WRAPPER"] }}'>
         <div class='row'>
             {% if current_user.has_role("admin") or current_user.has_role("finance") %}
-            <a href="{{ url_for('index') }}">
+	    <!--<a href="{{ url_for('index') }}">-->
```

Comment:
dead code

Reply:
i removed this yesterday but maybe after the code was checked.

Reply:
re-pushed without this in the index,.html